interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -162,6 +165,9 @@ void parseArgs(String[] args) {
           + "\n  --use_tls=true|false        Whether to use TLS. Default " + c.useTls
           + "\n  --use_alts=true|false       Whether to use ALTS. Enable ALTS will disable TLS."
           + "\n                              Default " + c.useTls
+          + "\n  --use_upgrade=true|false    Whether to use the h2c Upgrade mechanism. "
+          + "Enabling h2c Upgrade will disable TLS."
```

Comment:
We would want this to be on a new line and indented, to make it look fine even on non-super-wide terminals.